package com.accountmanagement.request;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class ForgotPasswordRequest {

    @NotBlank(message = "Email Field Is Required")
    @Email(message = "Invalid Email Format")
    private String email;
}
